Missing explicit instantiations of dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<1>

As evidenced by the testsuite when using clang/libc++ [1]:
```
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<1>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<1, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<1, double> const> > > const&, dealii::Point<1, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<1>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<1, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<1, double> const> > > const&, dealii::Point<1, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<1>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<1, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<1, double> const> > > const&, dealii::Point<1, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<1>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<1, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<1, double> const> > > const&, dealii::Point<1, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<2>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<2, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<2, double> const> > > const&, dealii::Point<2, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<2>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<2, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<2, double> const> > > const&, dealii::Point<2, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<2>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<2, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<2, double> const> > > const&, dealii::Point<2, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<2>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<2, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<2, double> const> > > const&, dealii::Point<2, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<3>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<3, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<3, double> const> > > const&, dealii::Point<3, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<3>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<3, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<3, double> const> > > const&, dealii::Point<3, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<3>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<3, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<3, double> const> > > const&, dealii::Point<3, double> const&)'
undefined reference to 'dealii::NonMatching::internal::QuadratureGeneratorImplementation::Definiteness dealii::NonMatching::internal::QuadratureGeneratorImplementation::pointwise_definiteness<3>(std::__1::vector<std::__1::reference_wrapper<dealii::Function<3, double> const>, std::__1::allocator<std::__1::reference_wrapper<dealii::Function<3, double> const> > > const&, dealii::Point<3, double> const&)'

```


[1] https://cdash.dealii.43-1.org/test/14963056

--- a/source/non_matching/quadrature_generator.inst.in
+++ b/source/non_matching/quadrature_generator.inst.in
@@ -36,6 +36,12 @@ for (deal_II_dimension : DIMENSIONS)
           find_extreme_values(
             const std::vector<FunctionBounds<deal_II_dimension>> &);
 
+          template Definiteness
+          pointwise_definiteness(
+            const std::vector<
+              std::reference_wrapper<const Function<deal_II_dimension>>> &,
+            const Point<deal_II_dimension> &);
+
           template void
           estimate_function_bounds(
             const std::vector<

